python/ELB_DELETION_PROTECTION_ENABLED/ELB_DELETION_PROTECTION_ENABLED.py

```diff
@@ -65,8 +65,9 @@
 def evaluate_compliance(event, configuration_item, valid_rule_parameters):
     for attribute in configuration_item['supplementaryConfiguration']['LoadBalancerAttributes']:
         # Scenario 2: deletion_protection.enabled is set to True in 'LoadBalancerAttributes' in the configuration item
-        if attribute['key'] == 'deletion_protection.enabled' and attribute['value'] == 'true':
-            return build_evaluation_from_config_item(configuration_item, 'COMPLIANT')
+        if attribute['key'] == 'deletion_protection.enabled':
+            if attribute['value'] == 'true':
+                return build_evaluation_from_config_item(configuration_item, 'COMPLIANT')
```

Comment:
You need to add a break there. Because the "else" will be a deletion_protection.enabled == to false.